x-pack/plugins/alerting/server/config.ts

```diff
@@ -20,6 +20,7 @@ export const configSchema = schema.object({
   maxEphemeralActionsPerAlert: schema.number({
     defaultValue: DEFAULT_MAX_EPHEMERAL_ACTIONS_PER_ALERT,
   }),
+  defaultRuleTaskTimeout: schema.string({ validate: validateDurationSchema, defaultValue: '10m' }),
```

Comment:
Would like to have a wider range of input on what a good default timeout is. Maybe bring up at the next sync?

Reply:
Decided to have '5m'